suite-common/wallet-utils/src/sendFormUtils.ts

```diff
@@ -146,7 +146,7 @@ export const prepareEthereumTransaction = (txInfo: EthTransactionData) => {
         chainId: txInfo.chainId,
         nonce: numberToHex(txInfo.nonce),
         gasLimit: numberToHex(txInfo.gasLimit),
-        gasPrice: numberToHex(toWei(txInfo.gasPrice, 'gwei')),
+        gasPrice: numberToHex(toWei(txInfo.gasPrice || '0', 'gwei')),
```

Comment:
In case of legacy, it should probably fail with `gasPrice` undefined? It will fail with `0` anyways.